pages/views/_layouts/content.njk

```diff
@@ -9,6 +9,7 @@
     <div class="content {{ containerCls or 'container' }} {{ 'container-aside' if aside }}">
       {% if title %}
         <h1 class="page-title">
+          {% if preview %}{% include "static/assets/" + preview %}{% endif %}
```

Comment:
Sorry but the `preview` for the icon looks strange, I meant that we could rename the field for examples collection.
If we are going to change that field, we can consider `sectionIcon`,  `collectionIcon`, or smth like that